src/main/java/appeng/items/storage/ItemBasicStorageCell.java

```diff
@@ -123,64 +124,82 @@ public void addCheckedInformation(final ItemStack stack, final EntityPlayer play
                 .getCellInventory(stack, null, StorageChannel.ITEMS);
 
-        if (inventory instanceof CellInventoryHandler handler) {
-            final CellInventory cellInventory = (CellInventory) handler.getCellInv();
-
-            if (cellInventory != null) {
-                lines.add(
-                        NumberFormat.getInstance().format(cellInventory.getUsedBytes()) + " "
-                                + GuiText.Of.getLocal()
-                                + ' '
-                                + NumberFormat.getInstance().format(cellInventory.getTotalBytes())
-                                + ' '
-                                + GuiText.BytesUsed.getLocal());
-
-                lines.add(
-                        NumberFormat.getInstance().format(cellInventory.getStoredItemTypes()) + " "
-                                + GuiText.Of.getLocal()
-                                + ' '
-                                + NumberFormat.getInstance().format(cellInventory.getMaxItemTypes())
-                                + ' '
-                                + GuiText.Types.getLocal());
-
-                if (handler.isPreformatted()) {
-                    String filter = cellInventory.getOreFilter();
-                    if (filter.isEmpty()) {
-                        final String list = (handler.getIncludeExcludeMode() == IncludeExclude.WHITELIST
-                                ? GuiText.Included
-                                : GuiText.Excluded).getLocal();
-
-                        if (handler.isFuzzy()) {
-                            lines.add(GuiText.Partitioned.getLocal() + " - " + list + ' ' + GuiText.Fuzzy.getLocal());
-                        } else {
-                            lines.add(GuiText.Partitioned.getLocal() + " - " + list + ' ' + GuiText.Precise.getLocal());
-                        }
-                        if (GuiScreen.isShiftKeyDown()) {
-                            lines.add(GuiText.Filter.getLocal() + ": ");
-                            for (int i = 0; i < cellInventory.getConfigInventory().getSizeInventory(); ++i) {
-                                ItemStack s = cellInventory.getConfigInventory().getStackInSlot(i);
-                                if (s != null) lines.add(s.getDisplayName());
-                            }
-                        }
-                    } else {
-                        lines.add(GuiText.PartitionedOre.getLocal() + " : " + filter);
-                    }
+        final CellInventoryHandler handler;
+        if (inventory instanceof CellInventoryHandler h) {
+            handler = h;
+        } else return;
 
-                    if (handler.getSticky()) {
-                        lines.add(GuiText.Sticky.getLocal());
-                    }
+        final CellInventory cellInventory = (CellInventory) handler.getCellInv();
+        if (cellInventory == null) {
+            return;
+        }
```

Comment:
You could do the same sort of instanceof as above here to avoid needing to unchecked cast to `CellInventory` (since null will fail any instanceof check anyway)